app/src/main/java/org/ole/planet/myplanet/utilities/Utilities.java

```diff
@@ -73,6 +75,16 @@ public static String formatDate(long date) {
         return datetime;
     }
 
+    public static String getformatedDate(String stringDate, String pattern) {
+        try {
```

Comment:
@yuviii move this function to ```TimeUtils.java```
https://github.com/open-learning-exchange/myplanet/blob/master/app/src/main/java/org/ole/planet/myplanet/utilities/TimeUtils.java

Reply:
@rrijal53 There are duplicate date functions in `Utilities` and `TimeUtils` making things confusing. Creating the new issues to remove the duplicate function from `Utilities` and `TimeUtils`.